go.mod

```diff
@@ -76,4 +75,4 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-go 1.19
+go 1.21
```

Comment:
I'm curious: Any particular reason to *not* use Go 1.22?

Reply:
linting doesnt support 1.22 yet so I just set this to 121 for now. we can upgrade to 1.22 if we want to.

Reply:
actually it wasnt linting, it was the trivy vulnerability scanner that doesnt support 1.22 yet